### ISSUE TITLE
"Cannot .delete() databases" error.

### ISSUE THREAD

**Issue by rh0** [beakerbrowser/webdb#18]
When calling `.delete()` from the browser I recieve:

_Error: Cannot .delete() databases outside of the browser environment. You should just delete the files manually._

I had a nagging feeling that I misconfigured something in my project (so level-browserify is pulling the wrong level), but after a bit of sleuthing, nothing obvious is standing out to me.

**Comment by pfrazee** [comment]
Oh hmm, that's happening in beaker?

**Comment by rh0** [comment]
Yeah.  I'm attempting to clear the index before I open the db:
```javascript
await webdb.delete()
await webdb.open()
```
`.open()` works fine (along with all of the other webdb functions I'm using), but when I add `.delete()` beaker throws the error.

**Comment by pfrazee** [comment]
Ok weird, I dont know how that function got incorrect but it's definitely a bug. We need a new implementation of `delete()`!

**Comment by rh0** [comment]
I'm starting to think this is an issue with level-browserify, although walking through their issue queue its a little hazey what the intent was with this module.  There is a browser field in level-browserify's package.json file pointing at [browser.js](https://github.com/Level/level-browserify/blob/master/browser.js).  It looks like that should be doing the business, but for whatever reason does not seem to be respected when running browserify.  I'm going to open an issue over there to see if I can get any insight.

In the meantime I've found that the addition of a browser field in WebDB's package.json can function as a workaround.  Specifically something like:
```json
  "browser": {
    "level-browserify": "level-js"
  }
```
Although that somewhat defeats the purpose of using level-browserify.

**Comment by rh0** [comment]
This is indeed a problem with level-browserify. Confirmed this bug will be fixed once [this PR](https://github.com/Level/level-browserify/pull/31) is merged.

**Comment by pfrazee** [comment]
Nice

**Comment by rh0** [comment]
The PR referenced above was merged into v1.1.2 of level-browserify today.  Just updated modules and tested `delete()` again, and it seems to have fixed the issue!

**Comment by pfrazee** [comment]
Ok great, I bumped the dep explicitly in webdb and published `4.1.1`. Thanks for tracking that.
